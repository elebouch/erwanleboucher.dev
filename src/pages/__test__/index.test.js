import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import Index from "../index"

describe("Index", () => {
  it("renders correctly", () => {
    const { container } = render(<Index />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
