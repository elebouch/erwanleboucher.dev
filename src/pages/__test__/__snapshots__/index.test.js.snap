// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
.c1 {
  width: 100%;
  height: 3px;
  background-color: var(--dark);
  margin: 10px 0;
  -webkit-animation: reveal 0.7s forwards;
  animation: reveal 0.7s forwards;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  color: var(--dark);
  -webkit-animation: fadein 0.5s;
  animation: fadein 0.5s;
}

.c2 {
  font-family: "Archivo Black",sans-serif;
  font-weight: 400;
  font-size: 16px;
  margin-bottom: 10px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 10px;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c6:hover {
  -webkit-filter: grayscale(1);
  filter: grayscale(1);
}

.c6 > svg {
  color: var(--dark);
  padding: 10px;
  height: 50px;
  -webkit-transition: opacity 0.2s linear;
  transition: opacity 0.2s linear;
  cursor: pointer;
}

.c6 > svg:hover {
  opacity: 0.7;
}

.c4 {
  border-radius: 100%;
  object-fit: contain;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-bottom: 20px;
}

<article>
  <div
    class="c0"
  >
    <div
      class="c1"
    />
    <span
      class="c2"
    >
      About
    </span>
    <div
      class="c3"
    >
      <div
        class="c4 gatsby-image-wrapper"
        style="position: relative; overflow: hidden; display: inline-block;"
      >
        <picture>
          <source />
          <img
            alt=""
            loading="lazy"
            style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;"
          />
        </picture>
        <noscript>
          &lt;picture&gt;&lt;source srcset="undefined" /&gt;&lt;img loading="lazy" src="" alt="" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
        </noscript>
      </div>
    </div>
    <p>
      I am a French software developer, currently living in Berlin, Germany. I am a passionate developer, loving to write code in JavaScript, Go and Python, along being curious about web-design and DevOps.
    </p>
    <p>
      Enthusiast about anything related to new technologies, science or engineering, I am as well interested in motorsport, especially Formula 1 and Sci-fi.
    </p>
    <p>
      You can find me there:
    </p>
    <div
      class="c5"
    >
      <a
        aria-label="Github"
        class="c6"
        href="https://github.com/eleboucher"
        rel="noopener noreferrer"
        size="50"
        target="_blank"
      >
        <test-file-stub
          alt="GitHub"
          loading="lazy"
        />
      </a>
      <a
        aria-label="LinkedIn"
        class="c6"
        href="https://www.linkedin.com/in/erwan-leboucher/"
        rel="noopener noreferrer"
        size="50"
        target="_blank"
      >
        <test-file-stub
          alt="LinkedIn"
          loading="lazy"
        />
      </a>
      <a
        aria-label="Mail"
        class="c6"
        href="mailto:erwanleboucher@gmail.com"
        rel="noopener noreferrer"
        size="50"
        target="_blank"
      >
        <test-file-stub
          alt="Mail"
          loading="lazy"
        />
      </a>
      <a
        aria-label="Twitter"
        class="c6"
        href="https://twitter.com/elebouch"
        rel="noopener noreferrer"
        size="50"
        target="_blank"
      >
        <test-file-stub
          alt="Twitter"
          loading="lazy"
        />
      </a>
      <a
        aria-label="Sourcerer"
        class="c6"
        href="https://sourcerer.io/eleboucher"
        rel="noopener noreferrer"
        size="50"
        target="_blank"
      >
        <test-file-stub
          alt="Sourcerer"
          loading="lazy"
        />
      </a>
      <a
        aria-label="Sourcerer"
        class="c6"
        href="https://dribbble.com/eleboucher"
        rel="noopener noreferrer"
        size="50"
        target="_blank"
      >
        <test-file-stub
          alt="Dribbble"
          loading="lazy"
        />
      </a>
    </div>
    <p>
      You can also find my resume bellow:
    </p>
    <a
      aria-label="resume"
      class="c6"
      href="/resume.pdf"
      rel="noopener noreferrer"
      size="50"
      style="margin-left: 10px;"
      target="_blank"
    >
      <test-file-stub
        alt="resume"
        loading="lazy"
      />
    </a>
  </div>
</article>
`;
